plugin/src/main/java/org/jenkinsci/plugins/workflow/cps/CpsFlowExecution.java

```diff
@@ -267,6 +267,7 @@ public class CpsFlowExecution extends FlowExecution implements BlockableResume {
      *
      * @see #runInCpsVmThread(FutureCallback)
      */
+    @SuppressFBWarnings(value = "PA_PUBLIC_PRIMITIVE_ATTRIBUTE", justification = "Preserve API compatibility.")
```

Comment:
This is not part of any API. Could be at least package-scoped and https://github.com/jenkinsci/workflow-cps-plugin/blob/305192405bc0d5f8923aa75226e97a700aef4c5c/plugin/src/test/java/org/jenkinsci/plugins/workflow/WorkflowTest.java#L150 just deleted or something.
```suggestion
    @SuppressFBWarnings(value = "PA_PUBLIC_PRIMITIVE_ATTRIBUTE", justification = "TODO clean up")
```